src/contexts/MintBurnFormContext/MintBurnFormContext.tsx

```diff
@@ -436,20 +436,16 @@ export const MintBurnFormProvider: React.FunctionComponent<MintBurnFormProviderP
     }
 
     if (!isUndefined(margin)) {
-      try {
-        const hasEnoughFunds = await hasEnoughUnderlyingTokens(
-          positionAmm || poolAmm,
-          margin,
-          mode === MintBurnFormModes.ROLLOVER ? position : undefined,
-        );
-        if (!hasEnoughFunds) {
-          valid = false;
-          if (touched.current.includes('margin')) {
-            err['margin'] = 'Insufficient funds';
-          }
+      const hasEnoughFunds = await hasEnoughUnderlyingTokens(
+        positionAmm || poolAmm,
+        margin,
+        mode === MintBurnFormModes.ROLLOVER ? position : undefined,
+      );
+      if (!hasEnoughFunds) {
+        valid = false;
+        if (touched.current.includes('margin')) {
+          err['margin'] = 'Insufficient funds';
```

Comment:
Has the catch block here been removed in favour of the Sentry error model?

Reply:
no not really @Sol3iman if you have a closer look it seems that hasEnoughUnderlyingTokens already has a try catch... So having try catch here as well is redundant since the catch will never be executed. Kinda like dead code.

Reply:
Oh I see, the if clause handles that. Interesting. In that case this PR LGTM